airflow-core/src/airflow/api_fastapi/core_api/routes/public/dag_stats.py

```diff
@@ -74,15 +74,18 @@ def get_dag_stats(
     query_result = session.execute(dagruns_select)
 
     result_dag_ids = []
+    dag_display_names: dict[str, str] = {}
     dag_state_data = {}
-    for dag_id, state, count in query_result:
+    for dag_id, state, dag_display_name, count in query_result:
         dag_state_data[(dag_id, state)] = count
-        if dag_id not in result_dag_ids:
+        if (dag_id) not in result_dag_ids:
```

Comment:
Just a small nit, more like my personal preference.

```suggestion
        if dag_id not in result_dag_ids:
```

Reply:
Thanks, that would better.